src/Tickets/Admin/Onboarding/Tickets_Landing_Page.php

```diff
@@ -248,15 +246,15 @@ public function admin_content_checklist_section(): void {
 						tribe_classes(
 							[
 								'step-list__item' => true,
-								'tec-tickets-onboarding-step-2' => true,
-								'tec-admin-page__onboarding-step--completed' => isset( $completed_tabs[2] ) || ! empty( tribe_get_option( 'defaultCurrencyCode' ) ),
+								'tec-tickets-onboarding-step-1' => true,
```

Comment:
Note: a lot of this logic is guesses as the flow and designs weren't finalized yet. Needs a rework.